functions-and-scope.js

```diff
@@ -16,6 +16,19 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+function amountCumLaude  ( val ){
+    let counter = null;
```

Comment:
Het is beter om hier meteen 0 van te maken, omdat je hier immers een nummer gaat bijhouden
`let counter = 0;`